recommenders/models/tfidf/tfidf_utils.py

```diff
@@ -239,13 +241,17 @@ def __create_full_recommendation_dictionary(self, df_clean):
         # Similarity measure
         cosine_sim = linear_kernel(self.tfidf_matrix, self.tfidf_matrix)
 
+        # _sorted_idx has the indices that would sort the array.
+        _sorted_idx = np.argsort(cosine_sim, axis=1)
+
+        _data = list(df_clean[self.id_col].values)
+        _len_df_clean = len(df_clean)
```

Comment:
this is a minor thing, but in the repo we normally not use the notation `_data` or `_sorted...`, would you mind to change all those entries to `data`, `sorted_idx` or `len_df_clean`

Reply:
Sure, thanks for the feedback. I will update them.